easybuild/easyconfigs/r/R/R-3.2.3-intel-2016a-libX11-1.6.3.eb

```diff
@@ -0,0 +1,453 @@
+name = 'R'
+version = '3.2.3'
+
+libx11 = 'libX11'
+libxver = '1.6.3'
+versionsuffix = '-%s-%s' % (libx11, libxver)
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s']
+
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=yes --enable-R-shlib"
+#Actually use Tcl/Tk
+configopts += ' --with-tcl-config=$EBROOTTCL/lib/tclConfig.sh --with-tk-config=$EBROOTTK/lib/tkConfig.sh '
+# Enable graphics capabilities for plotting.
+configopts += " --with-cairo --with-libpng --with-jpeglib --with-libtiff"
+# some recommended packages may fail in a parallel build (e.g. Matrix), and we're installing them anyway below
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+    (libx11, libxver),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('libpng', '1.6.21'),  # for plotting in R
+    ('libjpeg-turbo', '1.4.2'),  # for plottting in R
+    ('LibTIFF', '4.0.6'), # for plotting in R
+    ('cairo', '1.14.6', versionsuffix), # for plotting in R
+    ('Java', '1.8.0_72', '', True),  # Java bindings are built if Java is found, might as well provide it
+    ('Tcl', '8.6.4'),  # for tcltk
+    ('Tk', '8.6.4', versionsuffix),  # for tcltk
+    ('cURL', '7.47.0'),  # for RCurl
+    ('libxml2', '2.9.3', '-Python-2.7.11%s' % versionsuffix),  # for XML
```

Comment:
@boegel Not, but `cairo` have a `GLib` dependency, and `GLib` have a `libxml2-Python` dependency. So all the chain should have `libxml2-Python` dependency to avoid conflicts (or we have to delete the `Python` dependency of the existing `GLib` easyconfig.